app/views/admin/poll/booths/index.html.erb

```diff
@@ -11,6 +11,8 @@
 <% end %>
 
 <% if @booths.any? %>
+  <%= render "/admin/shared/booth_search", url: admin_booths_path %>
```

Comment:
IMHO the search form should be displayed even if there are no booths. Otherwise, after searching for a term which doesn't return any results, it will be hard to change the search term. So this line probably works better if it's added just right before the line saying `<% if @booths.empty? %>`.